gnd/src/main/java/com/google/android/gnd/ui/home/featuredetails/FeatureDetailsViewModel.java

```diff
@@ -37,39 +37,41 @@
 public class FeatureDetailsViewModel extends ViewModel {
 
   @Hot
-  private final FlowableProcessor<Optional<Feature>> selectedFeature =
+  private final FlowableProcessor<Optional<Feature>> featureFlowable =
```

Comment:
I think a more consistent way to handle the sync vs async selectedFeature states might be to define a `LiveData<Boolean> moveMenuOptionVisible` and expose that to the view via a public access. If/when data binding on menu options is supported it will then be easy to switch. We could then get rid of the new `selectedFeature` member and restore this stream's original name. Wdyt?

Reply:
Yes, data binding isn't supported for options menu. So should this suggestion be added as a TODO?